admin/settings/class-admin-apple-settings-section.php

```diff
@@ -273,8 +273,10 @@ public function render_field( $args ) {
 			// Use select2 only when there is a considerable amount of options available.
 			if ( count( $type ) > 10 ) {
 				$field = '<select class="select2 standard" id="%s" name="%s' . $multiple_name . '" ' . $multiple_attr . ' size="%s">';
-			} else {
+			} elseif ( ! empty( $multiple_attr ) ) {
```

Comment:
Is this still necessary on the latest version of `v2.1.0`? I put in a fix for the wonky select menus in the settings section. If so, no worries, but it may no longer be needed.

Reply:
probably not. I'll double check